api/src/main/scala/com/gu/adapters/store/Store.scala

```diff
@@ -142,11 +142,12 @@ case class Dynamo(client: DynamoDbClient, fs: FileStore, props: DynamoProperties
 
     for {
       qr <- result.map(_.lastEvaluatedKey())
+      hasMore = !qr.isEmpty
```

Comment:
lastEvaluated key returns the value after which it stopped if there were more items than the pagesize.
If this is returned as empty it means there's no more pages to fetch.
For more information see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.Pagination.html

It looks like there should be a more intuitive way to do this with this java sdk that hides these things behind a simple iterator interface. We might consider refactoring in the future but for now I'm glad just fixing the bug